package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@schibstedspain/openads-connector-api",
+  "version": "1.0.0",
+  "description": "OpenAds Interfaces to be compiant developing a new OpenAds connector",
```

Comment:
Typo:

_"description": "OpenAds Interfaces to be compliant when developing a new OpenAds connector",_

Reply:
Fixed! thanks 👍